factory/django.py

```diff
@@ -266,6 +385,82 @@ def _make_data(self, params):
         return thumb_io.getvalue()
 
 
+def dependency_insert_order(data):
+    """This is almost the same function from django/core/serializers/__init__.py:sort_dependencies with a slight
+    modification on `if hasattr(rel_model, 'natural_key') and rel_model != model:` that was removed, so we have the
+    REAL dependency order. The original implementation was setup to only write to files in order if they had a know
```

Comment:
```suggestion
    REAL dependency order. The original implementation was setup to only write to fields in order if they had a known
```

I assume this reflects what you meant in this docstring better. Please, correct me if I'm wrong.